Realm/RLMResults.mm

```diff
@@ -295,25 +296,24 @@ - (id)objectAtIndexedSubscript:(NSUInteger)index {
     return [self objectAtIndex:index];
 }
 
--(id)minOfProperty:(NSString *)property {
-    RLMResultsValidate(self);
-
-    if (self.count == 0) {
+template<typename Table>
```

Comment:
Table for the template parameter name is confusing. TableType might be an improvement.